### PR TITLE
intégration scss text 

### PR DESCRIPTION
**Pull request by Marie-Sail**

### CODE REVIEW

**Review by NemoOnTheNet — APPROVED** [review]

File: frontend/src/components/homeText.jsx [hunk 1]
```diff
@@ -3,28 +3,28 @@ import "./homeText.scss";
 
 function HomeText() {
   return (
-    <>
-      <p>
-        Notre <span className="textColor">système Solaire</span> se trouve dans
-        un galaxie nommée <span className="textColor">Voie Lactée</span>
+    <div className="homeText">
+      <p className="text1">
```
Comment (Sax593) [inline frontend/src/components/homeText.jsx:7]: className bad naming

File: frontend/src/components/homeText.jsx [hunk 2]
```diff
@@ -3,28 +3,28 @@ import "./homeText.scss";
...
+        Le <span className="textColor">système Solaire</span> se trouve dans la
+        galaxie nommée <span className="textColor">Voie Lactée</span>
       </p>
-      <p>
-        Il est constitué d'une étoile (le soleil), de huit planètes (dont la
-        terre) ainsi que'une multitude de petit astre et météore.
+      <p className="text2">
```
Comment (Sax593) [inline frontend/src/components/homeText.jsx:11]: className bad naming

File: frontend/src/components/homeText.jsx [hunk 3]
```diff
@@ -3,28 +3,28 @@ import "./homeText.scss";
...
+        Il est constitué d'une étoile, de huit planètes ainsi que de multiple
+        petits astres, et météore.
       </p>
-      <p>
-        Chaque planètes
-        <span className="textColor">gravitent autour du soleil</span> sur leurs
+      <p className="text3">
```
Comment (Sax593) [inline frontend/src/components/homeText.jsx:15]: className bad naming

File: frontend/src/components/homeText.jsx [hunk 4]
```diff
@@ -3,28 +3,28 @@ import "./homeText.scss";
...
+        Les planètes
+        <span className="textColor"> gravitent autour du soleil</span> sur leurs
         propres orbites.
       </p>
-      <p>
+      <p className="text4">
```
Comment (Sax593) [inline frontend/src/components/homeText.jsx:20]: className bad naming

**Review by Sax593 — CHANGES_REQUESTED** [review]

**Review by NemoOnTheNet — APPROVED** [review]

**Review by Sax593 — APPROVED** [review]

**Pull request merged**